kratos/spatial_containers/bins_dynamic.h

```diff
@@ -802,23 +915,28 @@ class BinsDynamic : public TreeNode<TDimension,TPointType, TPointerType, TIterat
     BinsDynamic(BinsDynamic const& rOther);
 
 private:
+    ///@name Member Variables
+    ///@{
+
+    IteratorType mitPointsBegin;         /// Iterator to the first point
+    IteratorType mitPointsEnd;           /// Iterator to the last point
```

Comment:
Why the change of the name? we have never called iterators `mitXXXX`

Reply:
Yes, we have. But I can revert it if it is a problem

Reply:
Na, its just the first time i see it in Kratos S: